packages/calcite-components/src/components/flow/flow.tsx

```diff
@@ -134,51 +147,91 @@ export class Flow implements LoadableComponent {
     return this.setFocus();
   }
 
-  getFlowDirection = (oldFlowItemCount: number, newFlowItemCount: number): FlowDirection | null => {
-    const allowRetreatingDirection = oldFlowItemCount > 1;
-    const allowAdvancingDirection = oldFlowItemCount && newFlowItemCount > 1;
+  resetFlowDirection = (): void => {
+    if (this.el.hasAttribute("data-test-disable-animation-reset")) {
```

Comment:
Can you add this under a `Build.isTesting` block (see [example](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/utils/config.ts#L36)) so that it doesn't get included in production builds?